languages/java/exercises/concept/inheritance/.docs/introduction.md

```diff
@@ -0,0 +1,50 @@
+Inheritance is a core concept in OOPS (Object Oriented Programming). It donates IS-A relationship.
```

Comment:
I think OOPS is a typo, maybe it was OOP ?

Reply:
🤣  we should hide this comment conversation from the people out there. OOP vs OOPS is still an ambiguous topic. 
If you say, I'll make it OOP. Anyways, both of them mean the same!